.gitlab/functional_test/system_probe.yml

```diff
@@ -5,6 +5,31 @@
 # include:
 #   - /.gitlab/kitchen_common/testing.yml
 
+.pull_test_dockers:
+  image: 486234852809.dkr.ecr.us-east-1.amazonaws.com/ci/datadog-agent-buildimages/docker_x64:$DATADOG_AGENT_BUILDIMAGES
+  needs: []
+  tags: ["runner:docker"]
+  stage: functional_test
+  script:
+    # DockerHub login for build to limit rate limit when pulling base images
+    - DOCKER_REGISTRY_LOGIN=$(aws ssm get-parameter --region us-east-1 --name ci.datadog-agent.$DOCKER_REGISTRY_LOGIN_SSM_KEY --with-decryption --query "Parameter.Value" --out text)
+    - aws ssm get-parameter --region us-east-1 --name ci.datadog-agent.$DOCKER_REGISTRY_PWD_SSM_KEY --with-decryption --query "Parameter.Value" --out text | docker login --username "$DOCKER_REGISTRY_LOGIN" --password-stdin "$DOCKER_REGISTRY_URL"
+    # Pull base images
+    - mkdir -p /tmp/dockers
+    - pip install pyyaml
+    - inv -e system-probe.save-test-dockers --output-dir /tmp/dockers --arch $ARCH
+    - $S3_CP_CMD --recursive --exclude "*" --include "*.tar" /tmp/dockers $S3_ARTIFACTS_URI/kitchen-dockers/$ARCH
```

Comment:
If you just need S3 as temporary storage between two jobs, you can use Gitlab artifacts for that.

Reply:
tried to use artifacts
but seems like it overrides some other dependencies

Reply:
https://github.com/DataDog/datadog-agent/pull/14589/files/76cf8295ab2584adafb3c678ea53365a6a482e61..6945ff2e87a558590750631e31998d28603dca0d

Reply:
https://gitlab.ddbuild.io/DataDog/datadog-agent/-/jobs/204335169/raw

Reply:
Specifying `dependencies: ["pull_test_dockers_x64"]` means to only pull artifacts from those jobs, so it is turning off pulling artifacts from `tests_ebpf_*`. https://docs.gitlab.com/ee/ci/yaml/#dependencies

Reply:
fixed